account_setup.py

```diff
@@ -189,6 +189,7 @@ def create_aad_user(credentials, tenant_id, **kwargs):
                 display_name=display_name,
                 password_credentials=[
                     PasswordCredential(
+                        start_date=datetime(2000, 1, 1, 0, 0, 0, 0, tzinfo=timezone.utc),
```

Comment:
shoudn't that be current time?

Reply:
I don't think it should matter, It will have the same effect.

We shouldn't have to include this line at all, this is only necessary because there is a bug in the graphrbac client if start_date isn't specified.